testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/client/KeycloakTestingClient.java

```diff
@@ -42,6 +45,11 @@
         } else {
             ResteasyClientBuilder resteasyClientBuilder = new ResteasyClientBuilder();
             resteasyClientBuilder.connectionPoolSize(10);
+            if (serverUrl.startsWith("https")) {
+                // Disable PKIX path validation errors when running tests using SSL
+                HostnameVerifier hostnameVerifier = (hostName, session) -> true;
```

Comment:
Isn't this the same as hostnameVerification(ResteasyClientBuilder.HostnameVerificationPolicy.WILDCARD) ?

Reply:
Yes, fixed. Thanks!